### ISSUE TITLE
Scaling-/translate3d Problem with Firefox/Safari

### ISSUE THREAD

**Issue by dacbreakpoint** [svgdotjs/svg.draw.js#53]
Hello together,
I am building a big application around the _svg.js eco-system_ with many other plugins like draw,resize,select, draggable.
Sadly I am stuck at some point especially by using **Firefox** or **Safari** Browser.

### Environment
The problem appears by wrapping the svg-layer with a div which uses the **scaling- and trandlate3d** functionality. By using the mouse-wheel or the mousedown-event in my prod-environment the scaling-/translate3d is used to emulate panning-/zooming behavior. I dont want to use the panzoom-plugin because the svg-layer is just one part of a bigger environment which also should be scaled.

### Problem
If you draw a rectangle or resize the object with the Firefox-Browser the cursor doesnt stick to the selected corner - it differs from the rectangle. I am unable to discover the main cause but I have found out that the screenCTM-Matrix didn't get changed in Firefox.
It works for all other browsers like: Chrome, Chromium, Opera, Edge even with Internet-Explorer

For better understanding I created a _minimal setup_ to present the bug: [JSFiddle-Snippet](https://jsfiddle.net/pw4a2qb0/)

### Questions
- Is there a way to fix the bug?
- Is this a browser- or plugin related bug?

Thank you in advance for your help

**Comment by Fuzzyma** [comment]
The browser behavior regardig creenCTM is different across browsers. Actually we implemented a workaround for that but it seems it doesnt always work.
You should run simple tests around that issue to see where the problem comes from and maybe report a bug to browser vendors

**Comment by dacbreakpoint** [comment]
Thank you for the answer.

Can we solve the problem together or as a community? Maybe you can give us help or some advice which tests we can run. Sadly I am not a svg specialist so it would be nice to get some tips.

As workaround you mean the svg-helping points and matrix transformations?

// Edit: I currently saw that the problem only appears when I am using the "transform:scale" functionality.

**Comment by Fuzzyma** [comment]
I dont have time atm for doing this. So unfortunately I cant be of much help here.
We implemented a workaround in the library.
I never looked into 3d transform of the css spec mixed with svg transformations. Its a tedious mess.
The problem which we worked around in the library was for screenCTM of the `<svg>` element itself. I have no idea if it also helps for this issue - it seems now. So it mighth be a seperate bug.

For testing just create examples with transformations and get the screenCTM and see if it differs across browsers
